internal/server/presentation/log.go

```diff
@@ -59,6 +71,28 @@ func (h *AMQPLogHandler) HandleLog(msg amqp.Delivery) {
 	h.SendResponse(utils.OK, "OK", msg.ReplyTo, msg.CorrelationId)
 }
 
+// TODO: [Server] Improve the current RPC Implementation to reduct frontend delays
+// https://github.com/okuda-seminar/log_service/issues/85
+func (h *AMQPCTRLogHandler) HandleCTRLog(msg amqp.Delivery) {
+	req, err := ParseAMQPCTRLog(msg)
+	if err != nil {
+		h.SendResponse(utils.INVALID_ARGUMENT, fmt.Sprintf("Failed to parse log request: %v", err), msg.ReplyTo, msg.CorrelationId)
```

Comment:
For now, please make it return an error.

Regarding the queue, we want to control it using FlowControl, so an Ack should only be returned if there is no error.